src/ldp/http/BasicTargetExtractor.ts

```diff
@@ -10,16 +10,17 @@ import { TargetExtractor } from './TargetExtractor';
  * TODO: input requires more extensive cleaning/parsing based on headers (see #22).
  */
 export class BasicTargetExtractor extends TargetExtractor {
-  public async canHandle(input: HttpRequest): Promise<void> {
+  public async canHandle(): Promise<void> {
+    // Can handle all URLs
+  }
+
+  public async handle(input: HttpRequest): Promise<ResourceIdentifier> {
     if (!input.url) {
       throw new Error('Missing URL.');
     }
```

Comment:
That should never happen though? Isn't that a mandatory field?

Reply:
It's not actually

Reply:
Although the description does say that it will always occur for requests coming from an `http.Server` so this will probably never be an issue.